packages/ChangesReloaded-Core.package/CRChangeListItem.class/instance/className.st

```diff
@@ -0,0 +1,6 @@
+accessing
+className
```

Comment:
`className` is a method of `Object` that should always return the name of the class on which it is called (in this case, it would be `CRChangeListItem`) – we don't know which other parts of the system depend on this behavior (maybe the debugger does or some code coverage tools, I got no idea).
Anyways, we had the same issue in our code and changed it to `affectedClassName` (or was it `nameOfAffectedClass`?). I believe this would be applicable here too.

Reply:
`nameOfAffectedClass` it is